Curs09/C/02-max_struct.c

```diff
@@ -0,0 +1,81 @@
+﻿/* Programarea Calculatoarelor CA, 2022-2023
+*
+*	Crearea unei structuri cu 2 campuri (nume si nota) si 
+*		afisarea numelui cu cea mai mare nota. 
+*
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define NAMES_COUNT 5
+#define MAX_NAME_LENGTH 8
+
+struct Student
+{
+	char Name[MAX_NAME_LENGTH];
+	float Grade;
+};
+
+struct Student* get_random_students(const char names_array[][MAX_NAME_LENGTH])
```

Comment:
please find a proper name for this function. Something like `generating_random_grades`